tests/Unit/Adapter/MailTemplate/MailTemplateTwigRendererTest.php

```diff
@@ -64,43 +62,6 @@ public function testConstructor()
         $this->assertNotNull($generator);
     }
 
-    public function testFileNotFound()
```

Comment:
I get this test was removed because of the previous try / catch removal. It would be better not to remove it if possible.

Reply:
Yes. It needs to be removed.

If there is a scenario where the file really cannot be found, e.g. by wrong inputs, then the test should test exactly that.